api/HashChangeEvent.json

```diff
@@ -121,22 +121,22 @@
               "version_added": false
             },
             "opera": {
-              "version_added": true
+              "version_added": "15"
```

Comment:
Support for the event interface but none of the properties seemed suspect so I tested Opera 12.16 a bit. `document.createEvent('HashChangeEvent').newURL` is the empty string meaning this was supported in Presto.

Reply:
Oh, and actually https://staging-dot-mdn-bcd-collector.appspot.com/tests/api/HashChangeEvent can test this now with your recent custom test additions.